js/main.js

```diff
@@ -5,8 +5,8 @@ var userSettings = {};
 
 var VIEWER_VERSION = getManifestVersion(); // Current version
 
-const INNER_BOUNDS_WIDTH  = 1340;
-const INNER_BOUNDS_HEIGHT = 900;
+const INNER_BOUNDS_WIDTH  = 800;
+const INNER_BOUNDS_HEIGHT = 600;
```

Comment:
We can add more parameters here. 

They are used here:
https://github.com/betaflight/blackbox-log-viewer/blob/556198c962795e67d09608c938a1d1b76105c7db/js/main.js#L114-L118

And the parameters accepted are here: https://docs.nwjs.io/en/latest/References/Manifest%20Format/#window-subfields

The most important is `id`, if we pass it, it will remember the latest position/size in theory, and it will use it for the new window.

We have one "id": "main" for the main window, but this, that are for additional windows, maybe we must have one id for them.

Something like :
```javascript
const ID_SECONDARY_WINDOW = 'ID_ SECONDARY_WINDOW';
...
 gui.Window.open(INITIAL_APP_PAGE, 
 {  
     'id' : ID_SECONDARY_WINDOW,
     'min_width'  : INNER_BOUNDS_WIDTH, 
     'min_height' : INNER_BOUNDS_HEIGHT, 
 }, 
```
I think a better solution will be to add a number to the id, like:
- 'ID_ SECONDARY_WINDOW_1'
- 'ID_ SECONDARY_WINDOW_2'
- 'ID_ SECONDARY_WINDOW_3'
- ...

In this way it will remember various secondary windows positions and size, and they will be always the same if we open more than 1. But maybe we can let this for a future.

Reply:
Looking at the Chrome window, if we use the same ID for all maybe it simply focuses the window created: https://developer.chrome.com/docs/extensions/reference/app_window/#type-CreateWindowOptions

So maybe we need to go to the _1, _2, _3 version. If you can test it we can know more.

Reply:
Thanks for your feedback ... I have no clue about JS or windows but will try :-)

Reply:
I made a change I think as suggested, but it makes no difference.
On opening the app, the initial window is at size and location of last window.
The new window opens centred at 800x600.

Reply:
It seems that with the code above, the values in main.js apply to opening a new window from an existing window, and it opens at the minimum size.
The values in package.json affect the minimum size of the initial window that opens on arming.
It seems that within main.js, setting the minimum works perfectly well.  The only issue is that when it opens, it opens at minimum size.  Perhaps we can put a different parameter for the initial width on opening?

Reply:
I added a width and height that operates on opening a secondary window.
This is working really well.
After starting the app, the initial window opens at the same location and size as on the last run of the app.  It can be resized down to 800x600. This is good.
When a secondary window is opened, it opens at the position of the most recent (previous) secondary window at 1000x760px.  It also can be resized as small as 800x600.  If it is closed, a new secondary window opens at the location of the previous secondary window.  This too is good. 
The only problem with the code now is that only one secondary window can be opened :-) the new window button doesn't work once a single secondary window is open.
So yes we need to add a number to the ID of the secondary window...
@McGiverGim could you please explain how to do that?  It will be great, I think, once that is done.